src/popup/utils/utils.ts

```diff
@@ -506,3 +507,13 @@ export function isTxOfASupportedType(encodedTx: string, isTxBase64 = false) {
   }
   return SUPPORTED_TX_TYPES.includes(SCHEMA.OBJECT_ID_TX_TYPE[txObject.tag]);
 }
+
+const hasValidationKey = (error: any, expectedKey: string) => (
+  error.validation
+  && error.validation.some(({ key }: any) => expectedKey === key)
+);
+
+export const isInsufficientBalanceError = (error: any) => (
+  error instanceof InvalidTxError
+  && hasValidationKey(error, 'InsufficientBalance')
+);
```

Comment:
Please use regular functions instead of arrows same as all other functions here.